v2.0/getting-started/docker/tutorials/advanced-policy.md

```diff
@@ -2,229 +2,292 @@
 title: Accessing Calico policy with Calico as a network plugin
 ---
 
-
 ## Background
 
-With Calico, a Docker network represents a logical set of rules that define the 
-allowed traffic in and out of containers assigned to that network.  The rules
-are encapsulated in a Calico "profile".
+With Calico as a Docker network plugin, Calico will create an identically named 
+[profile]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/profile)
+to represent each Docker network.  This profile is applied to each container 
+in that network and the profile is used by Calico to configure access policy
+for that container.  By default, the profile contains rules that allow full
+egress traffic but allow ingress traffic only from containers within the same
+network and no other source.
+
+> Custom policy for a network can be configured by creating in advance, or editing,
```

Comment:
I'd just move this into the previous paragraph as:

Users can manually choose different default rules by creating the profile in advance using Calico.